pipeline_dp/budget_accounting.py

```diff
@@ -123,7 +186,7 @@ def __init__(self, total_epsilon: float, total_delta: float):
 
         self._total_epsilon = total_epsilon
         self._total_delta = total_delta
-        self._mechanisms = []
+        super().__init__()
```

Comment:
please put it as a first instruction in` __init__()` (and the same in PLDAccountant)